doc/api/net.md

```diff
@@ -344,6 +347,7 @@ changes:
   * `ipv6Only` {boolean} For TCP servers, setting `ipv6Only` to `true` will
     disable dual-stack support, i.e., binding to host `::` won't make
     `0.0.0.0` be bound. **Default:** `false`.
+  * `signal` {AbortSignal} An AbortSignal that may be used to close a listening server.
```

Comment:
a docs example would be nice - namely mentioning this calls `.close` internally and doesn't stop ongoing connections the server is already serving.

Reply:
added